lib/setup-chromedriver.sh

```diff
@@ -5,13 +5,19 @@ set -eo pipefail
 VERSION=$1
 ARCH=$2
 
-if ! type -a google-chrome > /dev/null 2>&1; then
-    sudo apt-get update
-    sudo apt-get install -y google-chrome
+if [ "$ARCH" == "linux64" ]; then
+    if ! type -a google-chrome > /dev/null 2>&1; then
+        sudo apt-get update
+        sudo apt-get install -y google-chrome
+    fi
 fi
 
 if [ "$VERSION" == "" ]; then
-  CHROME_VERSION=$(google-chrome --version | cut -f 3 -d ' ' | cut -d '.' -f 1)
+    if [ "$ARCH" == "mac64" ]; then
```

Comment:
Actually, this will definitely fail under a Windows-build unless `$VERSION` is explicitly passed...

Reply:
```
$ChromePath = "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe"
[System.Diagnostics.FileVersionInfo]::GetVersionInfo($ChromePath).ProductVersion.Split('.')[0]
```

Can be used to extract the major version on Windows asking a few others to test. I am not sure if Chrome is included with the `windows-latest` build, or if Chocolatey installs to this awkward default location (Chrome x64 installs to x86).

Reply:
Going to leave this as is, just wanted to add `macos-latest` support, and I don't have access to a Windows machine to really test. Seems like using `window-latest` would have already thrown a similar error though.